README.md

```diff
@@ -6,7 +6,23 @@ This repository holds the kcp tests that tests against the publicly available  i
 * Golang installed. See [Installing Golang](https://golang.org/doc/install),the newer the better.
         * Ensure you install Golang from a binary release [found here](https://golang.org/dl/), not with a package manager such as `dnf`
 * golint installed. See [Installing golint](https://github.com/golang/lint#installation)
-* Have the environment variable `KUBECONFIG` set pointing to your cluster
+* Install [kubelogin](https://github.com/int128/kubelogin.git) and [KCP kubectl plugin](https://github.com/kcp-dev/kcp.git)
```

Comment:
@Phaow may be it is better to ask them to install using the plugins ? that will be easy for everyone i believe ?

Reply:
Good option. I'll also add the simple way that download the package directly. In addition, I plan to write the dockfile and build the images with the release versions it will be more easily to run the test.

Reply:
Addressed.